bcgov/cas-registration
# Fix: 890 duplicate user operator records

It was possible to create rows in the user_operator table with duplicate user-operator pairs. This PR adds a unique constraint so that is no longer possible.

To test in the app:
- A subsequent user should not be able to request access to the same operator twice

To test the constraint:
- Create a user_operator access request & then attempt to insert a new row in that table with the same user-operator pair. It should throw a unique constraint error
